core/templates/pages/topic-viewer-page/practice-tab/practice-tab.component.ts

```diff
@@ -54,13 +56,16 @@ export class PracticeTabComponent implements OnInit {
   clientWidth!: number;
   subtopicMasteryArray: number[] = [];
   questionsStatusCallIsComplete: boolean = true;
+  betaTopics = ['place-values'];
```

Comment:
This is meant to be a list of topicUrlFragments of all the topics that are in beta. Although the list works here, I wonder if it belongs here. Should it be moved to something like App.constants perhaps?

Reply:
@nithinrdy I don't think we should be adding the topic names directly in the code base to have them in beta mode, maybe it should be a thing admin/topic managers should be able to decide. there might be more topics apart from 'place-values' which can have practice questions.

Reply:
@Rijuta-s Maybe I would have to add a new property to the Topic schema in the backend, and add a checkbox in the topic editor page to mark a topic as 'in beta'? I'm a little lost as I've not worked with Oppia's backend much.

Does changing the Topic schema need migration code, similar to state migration?

Reply:
Just a note, please assign me explicitly if you want me to answer a question -- I actually missed Rijuta's comment entirely and assignment will help assure it doesn't get lost.

Isn't there a checkbox in each topic that says whether to enable practice questions or not? You should be able to use that to see whether the current topic is a beta one.

Also @nithinrdy just a thought, you might want to get a bit more experience with Opipa's backend generally and make sure you understand the overall flow of data from frontend to backend and vice versa. This will be quite important if you want to do any full-stack projects.

Reply:
@seanlip Here is what the Practice Questions Launch Plan doc states:
## Qualifications for when to get out of Beta (on per-topic level)
 - Has improved results page
 - Has translation enabled and at least 30 practice questions in that topic available in all P0 languages

I'm assuming a topic is in beta if either of these requirements aren't checked.

Reply:
"Improved results page" seems like a platform-level thing, right? So it's just the second one.

I think what I would suggest is to just code for now assuming that all questions are beta. It will take quite a while (possibly a year) to get questions fully translated, so we could look into that later IMO.

Reply:
/cc @dchen97  for info.

Reply:
> "Improved results page" seems like a platform-level thing, right? So it's just the second one.

Yes, my bad -- it's just the second one.

> I think what I would suggest is to just code for now assuming that all questions are beta.

I'm assuming you meant "assuming all topics' practice features are in beta".
If yes, then I can push a commit removing the beta topic check.

Reply:
> "Improved results page" seems like a platform-level thing, right? So it's just the second one.

Yes, my bad -- its just the second one.
> I think what I would suggest is to just code for now assuming that all questions are beta.

Got it!